From 02fbf61a257db3c59a00d77f3a01e6ad4d089a8e Mon Sep 17 00:00:00 2001
From: Jeff Vander Stoep <jeffv@google.com>
Date: Thu, 18 Jan 2018 08:55:02 -0800
Subject: [PATCH 6/8] Suppress denials for non-API access

avc: denied { read } scontext=u:r:priv_app:s0:c512,c768
tcontext=u:object_r:wifi_prop:s0 tclass=file
avc: denied { read } scontext=u:r:priv_app:s0:c512,c768
tcontext=u:object_r:net_dns_prop:s0 tclass=file

Bug: 72151306
Test: build
Change-Id: I4b658ccd128746356f635ca7955385a89609eea1
---
 private/priv_app.te | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/private/priv_app.te b/private/priv_app.te
index f4cfc173..d91d19d5 100644
--- a/private/priv_app.te
+++ b/private/priv_app.te
@@ -112,11 +112,13 @@ allow priv_app selinuxfs:file r_file_perms;
 
 read_runtime_log_tags(priv_app)
 
-# suppress denials when safetynet scans /system
+# suppress denials for non-API accesses.
 dontaudit priv_app exec_type:file getattr;
 dontaudit priv_app device:dir read;
 dontaudit priv_app proc_interrupts:file read;
 dontaudit priv_app proc_modules:file read;
+dontaudit priv_app wifi_prop:file read;
+dontaudit priv_app net_dns_prop:file read;
 
 ###
 ### neverallow rules
-- 
2.17.0

